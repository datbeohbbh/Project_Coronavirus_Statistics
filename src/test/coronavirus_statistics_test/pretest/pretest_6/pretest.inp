10 8
? 5
q 1 1 29
q 1 9 29
q 1 2 39
q 2 1 4
q 1 3 26
? 4 2
? 6