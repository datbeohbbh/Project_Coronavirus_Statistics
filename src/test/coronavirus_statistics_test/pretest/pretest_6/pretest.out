Query ? 5 :
The number of safe cities is 10

Query ? 4 2 :
The answer is the city with the index 3 
- Information of the city 3 -
-> Number of current patients : 26
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 1
Quantity: 4 people

